test/OWNERS

```diff
@@ -0,0 +1,29 @@
+reviewers:
```

Comment:
you may want to check this list again, many of those folks are not even at the company anymore. Some of them are now working for different components or other orgs within redhat.